web_environment_ribbon/data/ribbon_data.xml

```diff
@@ -6,7 +6,7 @@
 <!-- Add ribbon name default configuration parameter -->
 <record id="default_ribbon_name" model="ir.config_parameter">
     <field name="key">ribbon.name</field>
-    <field name="value"><![CDATA[TEST<br/>({db_name})]]></field>
+    <field name="value"><![CDATA[TEST<br/><sup>({db_name})</sup>]]></field>
```

Comment:
Why this? :thinking:

Reply:
The previous one was not really readable the second line was too close to the border...
This way it looks nice and clean (see also commit message)